sat/report.py

```diff
@@ -1,7 +1,7 @@
 #
 # MIT License
 #
-# (C) Copyright 2019-2023 Hewlett Packard Enterprise Development LP
+# (C) Copyright 2019-2024 Hewlett Packard Enterprise Development LP
```

Comment:
You're probably going to make this change eventually when working on the next Jira for sorting by more than one field at a time, but for now, the file hasn't changed, so copyright doesn't need to be updated.